src/main/java/mate/academy/hwspringbootintro/dto/CreateBookRequestDto.java

```diff
@@ -11,10 +14,17 @@
 @ToString
 @AllArgsConstructor
 public class CreateBookRequestDto {
+    @NotNull
```

Comment:
for String is better to use
```suggestion
    @NotBlank
```